### ISSUE TITLE
sendContact is not working

### ISSUE THREAD

**Issue by alpires** [open-wa/wa-automate-nodejs#689]
Hi Guys, sendContact function is not working: window.WAPI.getChat(...).sendContact is not a function
Some idea? 
Version of whatsapp web: 2.2031.4

**Comment by smashah** [comment]
@alpires new version. Will look into it by tomorrow

**Comment by smashah** [comment]
Fix incoming

**Comment by alpires** [comment]
@smashah some news about this issue?

**Comment by smashah** [comment]
@alpires I'm currently working on the docs so you can inject the fix using a patch key

https://gumroad.com/l/BTMt?tier=Patches%20Only (I'm just using this as the mechanism to patch OTA instead of releasing a new version. It is completely free).

PLEASE MAKE SURE TO ENTER YOUR CORRECT HOST ACCOUNT NUMBER IN THE CHECKOUT FORM.

If you don't know the host account number use this:

```javascript
const hostAccNo = await client.getHostNumber();
console.log('hostAccNo',hostAccNo)
```

**Comment by mikelustosa** [comment]
Not work sendContact..

**Comment by smashah** [comment]
Next version will have the patch built in

**Comment by tinyCoder32** [comment]
Thank you, is there any ETA?

**Comment by mikelustosa** [comment]
> Implemented in v2.0.0 [f8ed576](https://github.com/open-wa/wa-automate-nodejs/commit/f8ed576b9ad67682b4da64420775af64118046e6) @tinyCoder32 @alpires @mikelustosa @apnerve
> 
> You no longer need to use gumroad for patches. They will now be fetched from GitHub.

SendContact not working with https://raw.githubusercontent.com/open-wa/wa-automate-nodejs/master/src/lib/wapi.js

Is there any solution?

**Comment by smashah** [comment]
@mikelustosa sendContact is fixed with a patch, not in wapi. It works with v 2 of this library
